deployments/kubernetes/cni-daemonset/istio-cni-daemonset.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
I don't see why this is a Daemonset? There is nothing that needs to be on the same node as far as I can tell, we are using standard k8s APIs

Reply:
Repairer can run as a command line utility or as a daemon.
Since we already have cni daemonset it should run as a part of it.

Reply:
I don't have an issue with having this as a single controller, but I think there was some talk about using this as a general purpose daemonset and adding additional functionality later. @mandarjog ?

Reply:
Merged with the existing daemonset in the newest version.

Reply:
Adding a controller adds yet another component. Running cni-repairer as part of cni daemonset makes sense. It is solving a problem that was created by self.

Reply:
I removed it and merged it with the existing daemonset.